src/build/buildExample.js

```diff
@@ -37,6 +37,29 @@ async function build(srcPath) {
   return absPath;
 }
 
+async function buildOne(srcPath) {
```

Comment:
Awesome build config :D What's the intention of adding this in? Just one-off builds?